.github/workflows/e2e.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
           key: node_modules_${{ runner.os }}_${{ hashFiles('**/yarn.lock') }}
 
       - name: Install framework modules
-        run: yarn install --frozen-lockfile
+        run: yarn install --immutable && yarn postinstall ## TO DO: is this really gonna be the way?
```

Comment:
I don't think we need `&& yarn postinstall`

Reply:
Correct, Yarn will execute it on its own.
Also of note `--immutable` is enabled by default when running on a CI

Reply:
Thanks!

> Also of note --immutable is enabled by default when running on a CI

^^ do you know how it checks env?

Reply:
I do indeed, it uses `ci-info`s `isCI` check

https://github.com/yarnpkg/berry/blob/dd043a8ab1618c0b02e28b2b40743bd336d63437/packages/plugin-essentials/sources/index.ts#L3
https://github.com/yarnpkg/berry/blob/dd043a8ab1618c0b02e28b2b40743bd336d63437/packages/plugin-essentials/sources/index.ts#L91